src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2576,6 +2583,10 @@ class NodeUtils(object):
             for nnid in self.numa_nodes:
                 vnode_key = vnode_name + '[%d]' % nnid
                 vnode_resc_avail = vnode_list[vnode_key].resources_available
+                if (self.cfg['cgroup']['memsw']['enabled']
+                        and self.cfg['cgroup']['memsw']['manage_cgswap']):
+                    vnode_resc_avail['cgswap'] = (vnode_resc_avail['vmem']
+                                                  - vnode_resc_avail['mem'])
```

Comment:
Same as before, what if the vnode_resc_avail['vmem'] - node_resc_avail['mem'] results in negative value?

Reply:
It never does, by construction, but same comment -- if you think inserting a max(0,...) makes sense, I'm not opposed to it.